src/web/handlers.rs

```diff
@@ -92,6 +93,9 @@ pub async fn get_tiles(
     {
         Ok(response) => {
             // adM sometimes returns an invalid response. We don't want to cache that.
+            if response.tiles.is_empty() {
+                reply = HttpResponse::NoContent();
```

Comment:
These 204's are including a payload though, of the empty set `{"tiles":[]}`, which violates the RFC.

We should return no payload in this case but let's also ensure the client side's happy with that, cc @ncloudioj